webview/src/experiments/components/table/TableHead.tsx

```diff
@@ -102,6 +101,8 @@ export const TableHead = ({
 
   const selectedForPlotsCount = getSelectedForPlotsCount(rows)
 
+  const firstExpColumnCellId = headerGroups[0].headers[0].id
```

Comment:
Instead of relying on array indexes to find the first experiments column cell, I realized I could find the cell with the `id` of the cell inside `headerGroup`.